examples/private_set_intersection/module/rust/src/tests.rs

```diff
@@ -0,0 +1,64 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+use crate::proto::private_set_intersection::{GetIntersectionResponse, SubmitSetRequest};
+use assert_matches::assert_matches;
+use oak::grpc;
+use protobuf::well_known_types::Empty;
+use protobuf::RepeatedField;
+use std::collections::HashSet;
+use std::iter::FromIterator;
+
+const MODULE_CONFIG_NAME: &str = "private_set_intersection";
+
+#[test]
+fn test_set_intersection() {
+    simple_logger::init().unwrap();
+
+    let (runtime, entry_channel) = oak_tests::run_single_module_default(MODULE_CONFIG_NAME)
+        .expect("Unable to configure runtime with test wasm!");
+
+    let mut req = SubmitSetRequest::new();
```

Comment:
Perhaps these almost the same protobuf construction into grpc request stanzas can be refactored out into `oak_tests` in a future PR.

Reply:
I'm not sure that the protobuf message construction can be easily commonized, because each one is specific to the particular gRPC service definition (but maybe I'm not understanding what you're suggesting).  

I did wonder if there was some way to re-use the client-side code from #571, but the different environments (client code running in a Node uses a handle, test code running outside of the runtime uses a `ChannelRef`) make it awkward.

Reply:
I guess what I'm thinking is a more Rust idiomatic way to create the protobuf types would be preferable if supported, e.g. `protobuf_encode(RustSubmitSetRequest{values:vec![_]})`, then `protobuf_encode` could be refactored into `grpc_request` but this is all dependent on the protobuf crate providing a more naturally Rust interface which I'm guessing it doesn't.